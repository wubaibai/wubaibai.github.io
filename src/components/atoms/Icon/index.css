.icon {
	& path {
		transition: 0.3s;
		fill: var(--secondary);
	}

	&:hover path {
		fill: var(--acent);
	}
}
