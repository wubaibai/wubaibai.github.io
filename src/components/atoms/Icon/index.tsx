import React from 'react';

import style from './index.css';

const Icon: React.FC = ({ children }) => <span className={style.icon}>{children}</span>;

export default Icon;
