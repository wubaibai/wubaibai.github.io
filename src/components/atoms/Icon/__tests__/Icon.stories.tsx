import React from 'react';
import { storiesOf } from '@storybook/react';

import Icon from 'components/atoms/Icon';

const stories = storiesOf('atoms/Icon', module);

stories.add('__interactive', () => <Icon>A</Icon>, { jest: 'Icon' });
