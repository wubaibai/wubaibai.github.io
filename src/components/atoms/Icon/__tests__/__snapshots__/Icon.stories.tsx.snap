// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`__interactive 1`] = `
<DocumentFragment>
  <span
    class="icon"
  >
    A
  </span>
</DocumentFragment>
`;
